draft-irtf-cfrg-aegis-aead.md

```diff
@@ -151,7 +151,7 @@ informative:
         org: University of Hyogo
     date: 2023
 
-  JLD21:
+  JLD22:
     title: "Guess-and-Determine Attacks on AEGIS"
     rc: "The Computer Journal, vol 65, 2022(8), pp. 2221–2230"
     seriesinfo:
```

Comment:
`date: 2021` can also be updated to 2022.

Reply:
done